Xamarin.Forms.Core/Items/SelectableItemsView.cs

```diff
@@ -41,9 +44,10 @@ public object SelectedItem
 			set => SetValue(SelectedItemProperty, value);
 		}
 
+
 		public IList<object> SelectedItems
 		{
-			get => (IList<object>)GetValue(SelectedItemsProperty);
+			get => _selectedItems;
```

Comment:
This breaks binding for `SelectedItems`. If you want to see the break in action, open Control Gallery on Android or iOS, navigate to Collection View Gallery -> Selection Galleries -> Multiple Selection, Bound and tap the "Clear CV selection and add items 0 and 3" button. In the list, only items 0 and 3 should be selected. With these changes to SelectableItemsView, no items are selected.

Reply:
AFAICT, none of the changes in SelectableItemsView here are necessary to implement selection on UWP.